app/src/main/java/com/dicoding/picodiploma/sejiwaproject/features/match/previous/PreviousMatchFragment.kt

```diff
@@ -0,0 +1,103 @@
+package com.dicoding.picodiploma.sejiwaproject.features.match.previous
+
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.dicoding.picodiploma.sejiwaproject.R
+import com.dicoding.picodiploma.sejiwaproject.commons.api.ApiRepository
+import com.dicoding.picodiploma.sejiwaproject.commons.utils.invisible
+import com.dicoding.picodiploma.sejiwaproject.commons.utils.visible
+import com.dicoding.picodiploma.sejiwaproject.features.league.detail.page.PageViewModel
+import com.dicoding.picodiploma.sejiwaproject.features.match.previous.model.Matchs
+import com.google.gson.Gson
+import kotlinx.android.synthetic.main.fragment_next_match.*
+
+/**
+ * A placeholder fragment containing a simple view.
+ */
+class PreviousMatchFragment : Fragment(),
+    PreviousMatchView {
+    private lateinit var rvMatch: RecyclerView
+    private lateinit var pageViewModel: PageViewModel
```

Comment:
This variable is not really needed, should be to remove.

Reply:
resolved, thanks